src/Engine/Manager/RenderManager.cpp

```diff
@@ -260,45 +281,77 @@ void RenderManager::Render(World& world, const glm::mat4& translationMatrix, con
             }
         }
     }
+    }
+    }
 
     /// @todo Render skinned meshes.
     
     // Light the world.
-    {
-        PROFILE("Light the world");
+    { PROFILE("Light the world");
+    { GPUPROFILE("Light the world", Video::Query::Type::TIME_ELAPSED);
+    { //GPUPROFILE("Light the world", Video::Query::Type::SAMPLES_PASSED);
         LightWorld(world, viewMatrix, projectionMatrix, viewProjectionMatrix, renderSurface);
     }
+    }
+    }
+
     /*
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Variable 'name' is assigned a value that is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunreadVariable)